src/vorta/views/repo_tab.py

```diff
@@ -59,6 +57,13 @@ def __init__(self, parent=None):
         self.init_repo_stats()
         self.populate_from_profile()
 
+    def set_repos(self):
+        count = self.repoSelector.count()
+        for x in range(4, count):
```

Comment:
```suggestion
        # The repositories are listed after the 4th entry in the repoSelector
        for x in range(4, count):
```

Reply:
Good to add a comment here. These numbers is usually what we forget to change. Finding the index of the separator would work too, but maybe overkill.